src/Admin/CitizenProjectAdmin.php

```diff
@@ -393,6 +395,26 @@ protected function configureListFields(ListMapper $listMapper)
             ->add('postAddress.cityName', null, [
                 'label' => 'Ville',
             ])
+            ->add('skills', TextType::class, [
+                'label' => 'Compétences  recherchées',
+                'template' => 'admin/citizen_project/list_skills.html.twig',
+            ])
+            ->add('matchedSkills', null, [
+                'label' => 'Compétences matchées',
+            ])
+            ->add('nextActionDate', null, [
+                'label' => 'Date de la prochaine action',
+                'mapped' => false,
+                'query_builder' => function(CitizenActionRepository $er) {
+
+                    dump($er);
```

Comment:
:trollface:

Reply:
woops!
`Remg added the Work in progress label just now`

Reply:
:ok_hand: